src/frontend/context/BrevModulContext.ts

```diff
@@ -321,7 +323,7 @@ const [BrevModulProvider, useBrevModul] = createUseContext(() => {
                 barnasFødselsdager: barnBrevetGjelder.map(barn => barn.fødselsdato || ''),
                 datoAvtale: skjema.felter.datoAvtale.verdi,
                 behandlingKategori,
-                antallUkerSvarfrist: skjema.felter.antallUkerSvarfrist.verdi,
+                antallUkerSvarfrist: +skjema.felter.antallUkerSvarfrist.verdi,
```

Comment:
Dette er en enkel måte å gjøre om fra string til number på. Skal i teorien alltid være et positivt tall når man kommer hit, fordi mulige verdier er tom streng eller tall, og det blir kastet feil hvis det er en tom streng.
![image](https://user-images.githubusercontent.com/25459913/224677649-7b52648e-f18a-40a0-9b95-e917cb51a27d.png)

Reply:
Jeg tenker at `Number()` er mer lesbart, særlig for mindre erfarne frontendutviklere 🤔

Reply:
Godt innspill! Endrer 😄